substrate/telemetry/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub fn init_telemetry(config: TelemetryConfig) -> slog_scope::GlobalLoggerGuard
 					first_time: true,	// ensures that on_connect will be called.
 				}
 			).fuse()
-		).build().fuse(), o!()
+		).chan_size(262144).overflow_strategy(slog_async::OverflowStrategy::DropAndReport).build().fuse(), o!()
```

Comment:
What does this number mean?

Reply:
256kb of buffer length to store info in before dropping entries, I suppose, @pepyakin ?

Reply:
Yeah, sure. Can we leave a comment? Just a lil bit wary about magic numbers.